t/op/list.t

```diff
@@ -218,15 +228,20 @@ sub FETCH {$_[0]{fetched}++}
 sub empty {}
 tie my $t, "";
 () = (empty(), ($t)x10); # empty() since sub calls usually result in copies
-is(tied($t)->{fetched}, undef, 'assignment to empty list makes no copies');
+is(my $obj = tied($t)->{fetched}, undef, 'assignment to empty list makes no copies');
 
-# this was passing a trash SV at the top of the stack to SvIV()
-ok(($0[()[()]],1), "[perl #126193] list slice with zero indexes");
+{
+    no warnings 'void';
+    no warnings 'uninitialized';
```

Comment:
why not using a single line to disable the unwanted warnings? `no warnings qw{uninitialized void};`

Reply:
Won't work, as per other instances.